vm/src/stdlib/os.rs

```diff
@@ -99,7 +101,12 @@ bitflags! {
         const O_CREAT = 0o0000_0100;
         const O_EXCL = 0o0000_0200;
         const O_APPEND = 0o0000_2000;
-        const O_NONBLOCK = 0o0000_4000;
+        const O_NONBLOCK = libc::O_NONBLOCK as u32;
```

Comment:
Let's get all flags from `libc`

Reply:
I think if all constants are replaced by libc, the rust and python code should be closer and better.

Reply:
@palaviv Was the `FileCreationFlags` structure created as a sort of wrapper for libc, so that we could implement on other platforms without libc?

Reply:
No. I did not see the `OpenOptionsExt`. Looking now that is a better way to go.